File: src/main/java/requestreplyapi/entries/Entry.java
package requestreplyapi.entries;

import requestreplyapi.common.IAddress;

public class Entry implements IAddress
{
	private String destinationId;
	private int portNr;
	public Entry(String theDest, int thePort)
	{
		destinationId = theDest;
		portNr = thePort;
	}

	public Entry() {
	}

	public String dest()
	{
		return destinationId;
	}
	public int port()
	{
		return portNr;
	}

    @Override
    public String toString() {
        return "Entry{" +
                "destinationId='" + destinationId + '\'' +
                ", portNr=" + portNr +
                '}';
    }
}

File: ObjectRequestBrokerCS/ObjectRequestBrokerCS/requestreplyapi/entries/Entry.cs
namespace requestreplyapi.entries
{
	using IAddress = common.IAddress;

	public class Entry : IAddress
	{
		private string destinationId;
		private int portNr;
		public Entry(string theDest, int thePort)
		{
			destinationId = theDest;
			portNr = thePort;
		}

		public Entry()
		{
		}

		public virtual string dest()
		{
			return destinationId;
		}
		public virtual int port()
		{
			return portNr;
		}

		public override string ToString()
		{
			return "Entry{" +
					"destinationId='" + destinationId + '\'' +
					", portNr=" + portNr +
					'}';
		}
	}
}